features/contenteditable.yml

```diff
@@ -0,0 +1,9 @@
+name: Contenteditable
+description: The `contenteditable` attribute represents whether the element should be editable by the user. If so, the browser modifies its widget to allow editing.
+spec: https://html.spec.whatwg.org/multipage/interaction.html#contenteditable
+caniuse: contenteditable
+compat_features:
+  - api.HTMLElement.contentEditable
+  - api.HTMLElement.contentEditable.plaintext-only
```

Comment:
This key should be moved into its own `contenteditable=plaintext-only` feature because there's recent developer interest in this feature. See this interop proposal: https://github.com/web-platform-tests/interop/issues/704

Additionally, moving it out would make this feature baseline high (which it really should be).